### ISSUE TITLE
Potential packaging issue - version 2.7.0.1 does not match one in metadata

### ISSUE THREAD

**Issue by Ludar-Pavel** [JohnSnowLabs/spark-nlp#2062]
## Description
I have a CI pipeline running with Github Actions. Starting recently, it has begun failing on the spark-nlp installation step (via pip) with the following output:
```
Collecting spark-nlp
  Downloading spark-nlp-2.7.0-1.tar.gz (27 kB)
ERROR: Requested spark-nlp from https://files.pythonhosted.org/packages/26/7b/30735db49dc9e268362208bf2c3704f0dd6117d925d6490b2d86eeca6db4/spark-nlp-2.7.0-1.tar.gz#sha256=3b9f69da39c6a7f9eaf1e4ae3e51a764d87463567fdcb6b50894ec57e825fcc2 has different version in metadata: '2.7.0'
Error: Process completed with exit code 1.
```
## Expected Behavior
This should not crash.

## Current Behavior
This crashes.

## Possible Solution
Changing the version number to be consistent should fix it.

## Steps to Reproduce
Please find the minimal reproduction repo [here](https://github.com/Ludar-Pavel/spark-nlp-deployment-bug-reproduction/runs/1647229133).

## Context
This breaks CI.

## Your Environment
* Spark NLP version: 2.7.0
* Link to your project (if any): [Ludar-Pavel/spark-nlp-deployment-bug-reproduction](https://github.com/Ludar-Pavel/spark-nlp-deployment-bug-reproduction)

**Comment by maziyarpanahi** [comment]
Hi,
I used build number and sometimes it complains about build and metadata are not the same.
However, you can use it as follow:

pip install spark-nlp==2.7.0 

It’s also a best practice to not install something without a version for compatibility issues.

**Comment by maziyarpanahi** [comment]
This should be resolved in the 2.7.1 release https://github.com/JohnSnowLabs/spark-nlp/releases/tag/2.7.1
Please re-open if the issue persists.
